python/ray/util/accelerators/intel-gpu.md

```diff
@@ -0,0 +1,75 @@
+# Intel GPU on Ray
```

Comment:
Can you please update the existing documents to include information of Intel GPU? 
`doc/source/ray-core/tasks/using-ray-with-gpus.rst`
`python/ray/util/accelerators/accelerators.md`